packages/app/utils/__tests__/object.test.ts

```diff
@@ -0,0 +1,55 @@
+import { clearEmpties } from '../object'
+
+describe('clearEmpties', () => {
```

Comment:
On thing this doesn't test (and the tested function doesn't clear) is strings. E.g., I'm able to "bypass" the Howto schema's requirements for the abstract field by putting in 10 spaces. Is this intentional?

Reply:
Yes, that use case wouldn't be handled here. 10 spaces isn't really empty, since a space is a character.

The JSONSchema spec validates it as well, if that case needed to be caught we should handle it in the affected schema by using regex or something along those lines.

This particular clearEmpties is only to resolve "React JSONSchema Form" leaving JSON properties in the JSON that shouldn't be there.

Reply:
Makes sense! It seems like a distinction that I'll muddle up again, so if there are no objections I might write a test to ensure that it *doesn't* remove spaces, next time I'm in the codebase.